sanic/server.py

```diff
@@ -929,15 +929,11 @@ def serve(
 
     # Register signals for graceful termination
     if register_sys_signals:
-        _singals = (SIGTERM,) if run_multiple else (SIGINT, SIGTERM)
-        for _signal in _singals:
-            try:
-                loop.add_signal_handler(_signal, loop.stop)
-            except NotImplementedError:
-                logger.warning(
-                    "Sanic tried to use loop.add_signal_handler "
-                    "but it is not implemented on this platform."
-                )
+        if os.name == "nt":
```

Comment:
Let's define a constant and use that instead of string literal directly

Reply:
I've added a constant in server.py but did not change this in `skipif` of OS-dependent tests. Do note that `os.name` is the more coarse platform definition where all Windows are `"nt"`, where `sys.platform` uses many names for Windows (and quite confusingly, 64 bit Python on 64 bit Windows is still called `"win32"`).